### PR TITLE
Added a new video 

### PR DESCRIPTION
**Pull request by samyakbvs**
#324

### CODE REVIEW

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please put your image in the img/video folder.
otherwise LGTM

**Comment by samyakbvs** [comment]
@nikit19

**Review by nikit19 — APPROVED** [review]

**Comment by nikit19** [comment]
@samyakbvs  Good job !

**Comment by samyakbvs** [comment]
pls merge

File: _data/video.yml [hunk 1]
```diff
@@ -3,3 +3,8 @@
   title: About FOSSASIA
   img: FOSSASIA.jpeg
   author_name: Supun Tharinda Edirisuriya
+  
+- url: https://www.youtube.com/watch?v=iJBtZgYQk0U&feature=youtu.be
```
Comment (anshumanv) [inline _data/video.yml]: Change video url to - https://www.youtube.com/watch?v=iJBtZgYQk0U

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments.

**Comment by samyakbvs** [comment]
@nikit19

**Comment by samyakbvs** [comment]
done

**Comment by samyakbvs** [comment]
@anshumanv pls merge

**Review by anshumanv — APPROVED** [review]
LGTM

**Review by dilpreetsio — APPROVED** [review]

**Review by vaibhavsingh97 — APPROVED** [review]

**Comment by vaibhavsingh97** [comment]
@samyakbvs Please squash your commits into one.

**Comment by samyakbvs** [comment]
I won't be able to squash as I have other commits in between
so pls merge like this

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**